libs/libnx/nxfonts/Makefile.sources

```diff
@@ -325,3 +325,4 @@ clean:
 distclean: clean
 	$(call DELFILE, nxfonts_convert_*bpp.c)
 	$(call DELFILE, nxfonts_bitmaps_*.c)
+	$(call DELFILE, nxfonts_tom-thumb-4x6.c)
```

Comment:
let's change the generated file to  nxfonts_bitmaps_tom-thumb-4x6.c like other font?

Reply:
ok, done.